test/v2/controllers/V2DeparturesControllerSpec.scala

```diff
@@ -785,6 +787,14 @@ class V2DeparturesControllerSpec
     }
   }
 
+  def generateResponse(messageId: MessageId) =
+    MessageResponseWithBody(
+      messageId,
+      arbitrary[OffsetDateTime].sample.value,
+      DepartureDeclaration,
+      Some("<CC015C><test>testxml</test></CC015C>")
+    )
```

Comment:
You've a few places where you use a MessageResponseWithBody off of a Arbitrary[MessageId] - how about you change them to use an Arbitrary[MessageResponseWithBody] using this inside? Much cleaner.

Reply:
Agree. Will implement.